lib/xcode/install.rb

```diff
@@ -266,6 +267,20 @@ def verify_integrity(path)
       puts `/usr/sbin/spctl --assess --verbose=4 --type execute #{path}`
       $?.exitstatus == 0
     end
+
+    def hdiutil(*args)
+      result = IO.popen(['hdiutil', *args]).read
+      fail Informative, "Failed to invoke hdiutil." unless $?.exitstatus == 0
```

Comment:
Indeed - forgot to `close` it before reading the `exitstatus`.

Fixed that in another place (when `curl` was invoked) as well.